### ISSUE TITLE
Quark Engine does not list all APIs of my sample

### ISSUE THREAD

**Issue by cryptax** [quark-engine/quark-engine#147]
I am using a malicious sample of SandroRAT, sha256: `f82d6f24af2a4444c696c64060582d8aed6280da578c4dea3bb71bd6a11ebcf8`. You can [download it from Koodous if you want](https://koodous.com/apks?search=f82d6f24af2a4444c696c64060582d8aed6280da578c4dea3bb71bd6a11ebcf8).

This sample uses `com.esotericsoftware.kryonet.Client`, in particular the `connect()` method from that class.
But `quark -i` does not show any APIs from this... why?

I am using **Quark v21.3.2**.

Proof `com.esotericsoftware.kryonet.Client` exists: this is taken from the smali file: `./smali/com/esoterics
oftware/kryonet/Client.smali`

```smali
.class public Lcom/esotericsoftware/kryonet/Client;
.super Lcom/esotericsoftware/kryonet/Connection;

# interfaces
.implements Lcom/esotericsoftware/kryonet/EndPoint;
```

It is used in `net/droidjack/server/w`:

```smali
.class Lnet/droidjack/server/w;
.super Ljava/lang/Object;
...
   invoke-direct {v0}, Lcom/esotericsoftware/kryonet/Client;-><init>()V
   sput-object v0, Lnet/droidjack/server/Controller;->u:Lcom/esotericsoftware/kryonet/Client;
```

Quark's output: 

```
# quark -i -a ./sandr.apk  | grep esoter
#
```
It **does not see anything from com/esotericsoftware/kryonet package.**

**Comment by krnick** [comment]
Hi @cryptax ,

Thank you for reporting this issue, the current `-i` option only shows the usage of the Android native API. It is my mistake that not clearly pointed out in the help message. If you think it is useful not only to show Android native APIs but also to show the names of custom classes and methods, I can add it to other options.

Currently, our rules only take two Android native APIs for input, so we have not considered the custom method of the rules. However, we did test the custom method rules and it also worked well.

**Comment by cryptax** [comment]
>However, we did test the custom method rules and it also worked well.
Yes that's what I did, and it works fine.

As it works, maybe indeed adding also custom rules to the list of `-i` would be nice.

**Comment by krnick** [comment]
> > However, we did test the custom method rules and it also worked well.
> > Yes that's what I did, and it works fine.
> 
> As it works, maybe indeed adding also custom rules to the list of `-i` would be nice.

No problems, I am working on it 😄

**Comment by krnick** [comment]
Hello @cryptax ,

I have updated the latest version to "v21.3.4", where I put the usage of listing Android API and custom methods in the option `-i`.

Now quark can list the usage below:

## Method

1. Android native API `quark -a malware.apk -i native`
2. Android custom methods `quark -a malware.apk -i custom`
3. all `quark -a malware.apk -i all`

## Permission

`quark -a malware.apk -p`

Thank you for providing us this idea! 😄

**Comment by krnick** [comment]
Fixed it in version "v21.3.4"
